.github/workflows/sync-docs.yaml

```
@@ -0,0 +1,72 @@
+name: Sync docs to docs site repo
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+  workflow_dispatch:
+
+jobs:
+  config-sync:
+    name: Synchronize docs to docs site repo
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.merged }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        source_repo:
+          [
+            "slack-github-action",
+          ]
```

```suggestion
        source_repo:
          - slack-github-action
```

This can also be listed without as much order!